examples/map/even_odd/example.py

```diff
@@ -22,5 +22,5 @@ def my_handler(keys: list[str], datum: Datum) -> Messages:
 
 
 if __name__ == "__main__":
-    grpc_server = Mapper(handler=my_handler)
+    grpc_server = MapServer(mapper_instance=my_handler)
```

Comment:
```suggestion
    grpc_server = MapServer(my_handler)
```

do we really need the named param? this is a mandatory field anyways.